app/experimenter/nimbus-ui/src/components/AppErrorDialog/index.tsx

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+
+const AppErrorDialog = ({ error }: { error: Error }) => {
+  return (
+    <div className="bg-grey-20 flex items-center flex-col justify-center h-screen">
```

Comment:
Huh, I was going to go check this out in Storybook build from CI but it doesn't seem to have triggered with this PR? I'm guessing these classes aren't actually working, but I'm sure we can easily grab some from Bootstrap to make this look decent.

I actually didn't think about styling this state, might've broken it up into another issue as a est of 1-2. We could still do that (I could file it in the second "after setup" epic) if you'd like since the meat of the issue is the Sentry setup.

Also, is `Dialog` still a good name here if it's not a modal? Maybe it is since we're communicating to the user and it's technically a blocking message since they can't do anything else. 😛

Reply:
Storybook build skipped because I submitted this from my own fork - will close this PR and re-open from upstream repo. For styling, I think I started this before we [got bootstrap merged in](https://github.com/mozilla/experimenter/commit/47dfb6a0e770deffeed37b76391b37caa431c63e) so I kind of punted on it. Also could just rename "Dialog" to "Message" but not sure it matters overmuch